java/mapreduce/pom.xml

```diff
@@ -63,10 +63,6 @@
       <groupId>org.apache.hive</groupId>
       <artifactId>hive-storage-api</artifactId>
     </dependency>
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
-    </dependency>
```

Comment:
Could you spin off this as a separate PR? We prefer to keep one theme in one PR. Especially, we avoid mixing 1) bug fix, 2) improvement, 3) clean-up. This looks like some kind of clean-up which is irrelevant to your PR intention.

Reply:
The maven package command needed to run the benchmarks fails without it. Should I still make it a separate PR that we merge in first?

Reply:
What command did you try?

Reply:
`./mvnw clean package -Pbenchmark`, as stated in the benchmark documentation

Reply:
Ah actually it looks like I was using with `-Dmaven.test.skip`, but when I changed that to `-DskipTests` like you said it was fixed. Updated.